modules/ROOT/pages/_partials/gs-adding-users-proc.adoc

```diff
@@ -186,21 +200,21 @@ To delete users:
 
 == Managing Red Hat 3scale API Management Platform users
 
-Users created in  Red Hat Single Sign-On can login to the 3scale console, however the user's account is created in 3scale after that initial login. The new user is assigned the role `member` and no permissions set by default. The user role and permission can only be set once a user account is created in 3scale.
+A user created in Red Hat Single Sign-On can log in to the 3scale console; however, the user's account is only created in 3scale after that initial login. The new user is assigned the role `member` and by default does not have permissions set. The user role and permission can only be set once a user account is created in 3scale.
```

Comment:
ok, this isn't specific enough, user must be created in MI SSO, ping @briandooley